device/celestica/x86_64-cel_e1031-r0/sonic_platform/common.py

```diff
@@ -195,7 +185,7 @@ def write_txt_file(self, file_path, value):
         return True
 
     def is_host(self):
-        return os.system(self.HOST_CHK_CMD) == 0
+        return subprocess.call(self.HOST_CHK_CMD) == 0

```

Comment:
Should pass DEVNULL to stdout and stderr parameters to suppress the output message.
Replace with return subprocess.call(HOST_CHK_CMD, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL) == 0